.github/workflows/all.yml

```diff
@@ -55,7 +54,9 @@ jobs:
           go-version: 1.17.x
       - name: Install golangci-lint
         working-directory: /tmp
-        run: go install "github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANG_CI_VERSION"
+        run: |
+          GOLANG_CI_VERSION=$(head -n 1 "${GITHUB_WORKSPACE}/.golangci.yml" | sed 's/# //g')
```

Comment:
A tiny nitpick, and definitely a preference-based comment that you should feel free to disregard: for such a tiny transformation, I would find using the `tr -d "# "` command much clearer (it's exactly what it was made for after all) than `sed` substitution in this context.